components/employees/add-employee-form/index.tsx

```diff
@@ -68,6 +74,14 @@ const AddEmployee: FC<AddEmployeeProps> = ({ setOpenEmployeeForm }) => {
   const onSubmit = async (formValues: AddEmployeeFormDetail) => {
     try {
       await addEmployee(formValues).unwrap();
+      if (sendInvite) {
+        await sendPasswordLink({
+          email: formValues.email,
+          type: "INVITE",
+          local: LOCAL,
+        });
+        toast.success("Invite sent successfully");
+      }
       toast.success("Employee Added");
```

Comment:
It seems like there is a possibility of displaying two toasts one after another. Please check if there's a possibility to show 1 toast only for both message i.e conditionally. Otherwise displaying two toasts may seem weird.

Reply:
Tester needs two toasts